### ISSUE TITLE
Verify that legacy SMS services' current limits are honoured when we introduce new limits

### ISSUE THREAD

**Issue by yaelberger-commits** [cds-snc/notification-planning#954]
_Verify that legacy SMS services' current limits are honoured when we introduce new limits_

## Description

Run query to check each service for max # of SMS parts sent in a day and ensure their limit remains high enough for them to keep sending

- [x] Write and run script to find services near their limit. ([here](https://docs.google.com/spreadsheets/d/1lVTAruwDJuobushcFUbJLQqtwFwimPe_3RVKCBz6NMY/edit#gid=1409336591))
- [x] Decide how much extra space we want to give folks. (25%)
- [x] Put the changes into the database.
- [x] Run query again before launch to ensure the limits are still good enough.

## QA Steps

- [ ] all services sufficiently below (historically) their sms limits

**Comment by sastels** [comment]
Query / data
https://docs.google.com/spreadsheets/d/1lVTAruwDJuobushcFUbJLQqtwFwimPe_3RVKCBz6NMY/edit#gid=0

**Comment by adriannelee** [comment]
How much space do we want to give the clients who already go over the limit? 25%

Steve will continue working on this today.

The script should have no results when the changes are implemented

**Comment by sastels** [comment]
Database updated with appropriate sms limits. Running the query now, the only services that appear are our own test services and three CRA services that have not sent since the summer.

**Comment by sastels** [comment]
Perhaps for QA another dev could take a good look at the query as well as run it and see the results.

**Comment by adriannelee** [comment]
Steve M will QA today

**Comment by smcmurtry** [comment]
This looks good! I verified that the current service limits were correct for all we had changed (see added column [here](https://docs.google.com/spreadsheets/d/1lVTAruwDJuobushcFUbJLQqtwFwimPe_3RVKCBz6NMY/edit#gid=1186390415)).

I also re-ran the query Steve A had used, and as of today there are not active services that need their limit adjusted, according to our "25% cushion" criteria that Steve A developed.
